makefile

```diff
@@ -11,10 +11,10 @@ ifdef CHAPTER
 	 -c :wq \
 	 book.tex
 endif
-	julia --color=yes pull_julia_code.jl
-	lualatex book
-	pythontex book
-	biber book
+	-julia --color=yes pull_julia_code.jl && \
```

Comment:
What does the `-julia` vs `julia` do?

Reply:
the makefile will continue the recipe, even if the command itself fails (cf. [here](https://www2.ipp.mpg.de/~dpc/gmake/make_47.html))